src/commands/cloudmanager/commerce/bin-magento/app/config/dump.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommerceCliCommand = require('../../../../../../base-commerce-cli-command')
+const { getProgramId } = require('../../../../../../cloudmanager-helpers')
+const commonFlags = require('../../../../../../common-flags')
+const commonArgs = require('../../../../../../common-args')
+
+class AppConfigDumpCommand extends BaseCommerceCliCommand {
+  static strict = false
+
+  async run () {
+    const { args, flags, argv } = this.parse(AppConfigDumpCommand)
+
+    const programId = getProgramId(flags)
+    const configTypes = argv.slice(1)
```

Comment:
also... do we need a check here that the argv array length is > 1

Reply:
I don't believe so, if args are length 1, then configTypes with be an empty array and on line 31, options property will be `["-n"]` , which is the desired behavior.

Reply:
Ah... and argv can never be length 0. Got it. Sorry for the noise.